### ISSUE TITLE
Zika travel health notice is used to reproduce core content

### ISSUE THREAD

**Issue by patlaj** [gc-proto/health-1#7]
There's currently core content for Zika:

https://www.canada.ca/en/public-health/services/diseases/zika-virus.html

...as well as a Travel health notice for Zika:

https://travel.gc.ca/travelling/health-safety/travel-health-notices/152

The travel health notice's primary purpose seems to be to call attention to news about outbreaks. But it also reproduces much of the existing core content. To make it worse, they also update this page *before* the core content when there's new information to convey. 

This duplication of content and slow update of the core content is a major problem.

**Comment by lisafast** [comment]
Agree that it is duplicative. I think the purpose is to have a single page for everything travel-related, rather than general Zika stuff (Yes, most of everything about Zika comes from travelling but over time, that could change). It pulls together a single page view. 

All of the Country pages that have Zika now link directly to the notice - so that if something did happen suddenly, and you are planning a trip to that country, you do get to the freshest advice. Imagine it was gone - which of the many Zika pages should a country link to? 

I think the suggestion is to at least add it into the template, rather than have it stranded outside is a good one. What is the alternative suggestion?

**Comment by patlaj** [comment]
Right, I created the issue to keep track of the problem and capture discussions about how to address it.
